### ISSUE TITLE
When used with Elixir, only a few completions appear

### ISSUE THREAD

**Issue by jcowgar** [manateelazycat/lsp-bridge#790]
I am on Emacs 30.0.50, Elixir 1.15.7, OTP 26.1.2.

I have tried with Lexical and ElixirLS, both yield the same result. All modules I try to complete exhibit similar behavior. No module is complete.

Here is a screen shot:

![image](https://github.com/manateelazycat/lsp-bridge/assets/251686/5c6b098a-0c1f-48f2-bfa9-868b5463a1e7)

The String module of course has many, many methods.

**Comment by manateelazycat** [comment]
Please report bug follow https://github.com/manateelazycat/lsp-bridge#report-bug

I need log.

Thanks

**Comment by jcowgar** [comment]
I began to go though and reproduce the bug as state in the report bug section of the readme. In doing so, I created the simplest project to show it and it was not. It turns out that the problem seems to be two fold. 

1. When I select String from the initial completion list, it was bringing in a local string module, not the system string module. We probably shouldn't have a local String module. I can fix that.
2. Once I do get the initial completion, subsequent completions are taking 30-60 seconds to appear, which made me think that other modules are not completing.

I am trying to find a sequence that triggers the problem and will then include some more information and the log you requested.

I am going to close this issue with this comment and when I have some actionable data, I'll open a correct issue with the real problem and the data requested in the bug submission part of the readme.

Thanks!
